.github/workflows/build-pr.yml

```diff
@@ -27,3 +31,31 @@ jobs:
         run: make test-style
       - name: Run unit tests
         run: make test-coverage
+
+  canary-release:
+    runs-on: ubuntu-latest
+    needs: build
+    env:
+      AZURE_STORAGE_CONNECTION_STRING: "${{ secrets.AZURE_STORAGE_CONNECTION_STRING }}"
+      AZURE_STORAGE_CONTAINER_NAME: "${{ secrets.AZURE_STORAGE_CONTAINER_NAME }}"
+      VERSION: canary
+    if: ${{ github.event }} == "push" && github.ref == 'refs/heads/main'
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v2
+
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: '1.16'
+
+      - name: Build Helm Binaries
+        run: |
+          make build-cross
+          make dist checksum VERSION="${VERSION}"
+
+      - name: Azure CLI Action
+        uses: Azure/cli@1.0.4
```

Comment:
probably want to be using the rolling `v1` branch instead of a specific version.

Reply:
Fixed.